### ISSUE TITLE
How to print barcodes

### ISSUE THREAD

**Issue by chack1172** [RamyTalal/Label-Printer#41]
Hello,

I'm trying to print a barcode with Brother QL-820NWB using Escp but it always print a blank label. Is it possible to print it this way or should I use a template?

This is the code I'm using:
`$stream = stream_socket_client("tcp://{$printerIp}:9100", $errorNumber, $errorString);

$printer = new Printer(new Escp($stream));
$printer->addCommand(new Command\Barcode($barcode, 80, Command\Barcode::WIDTH_SMALL, 'code39', false, 2.5));

$printer->printLabel();
fclose($stream);`

**Comment by RamyTalal** [comment]
Hello,

Escp should work, but I haven't tested it with the QL-820NWB. Does printing simple text work?

**Comment by chack1172** [comment]
Yes, printing text works without problems, also QR code prints fine. Instead barcode prints nothing.

**Comment by RamyTalal** [comment]
> Yes, printing text works without problems, also QR code prints fine. Instead barcode prints nothing.

I'm not sure if printing barcodes even works. It's been a while since I have worked on this library.
